digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1581,6 +1607,7 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
       logger.error(s" There are no roadlinks to update  with names, rollbacking update ${project.id}")
```

Comment:
this log should also change as should say something like "..there is no road addresses to update, rollbacking update.."